.github/workflows/whitesource.yml

```
@@ -0,0 +1,33 @@
+name: Whitesource Unified Agent Scan - GO
+
+on:
+  pull_request:
+    branches: [ master* ]
+
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
```

```suggestion
    name: Build
    runs-on: ubuntu-latest
    timeout-minutes: 20
```

Default timeout is something like 6 hours. Better to have it fail sooner in case something isn't right than to sit around for 6  hours with an issue IMO.